### ISSUE TITLE
Project maintained?

### ISSUE THREAD

**Issue by lewismc** [kyllingstad/debian-fmilib#1]
Hi @viproma I see that this project has not seen activity for a while. I have a few questions...
1. Can the project be used in it's current state?
2. Did you ever publish a debian package?
3. Do you have documentation regarding the installation of the package?
Thanks for any insight.

**Comment by kyllingstad** [comment]
Hi! I'm sorry, the project is no longer maintained. However:

1. I think so. The repo contains a complete copy of FMI Library 2.0.3 plus a `debian/` directory that enables you to build it using standard Debian packaging tools. So if you're happy using that version, it should in theory work just fine.
2. Yes, you can find prebuilt packages on the [Releases](https://github.com/viproma/debian-fmilib/releases/) page.
3. There is no additional documentation, just follow the normal Debian package installation procedure.

Note: In case you are not aware, we are not affiliated with FMI Library in any way. This repo was just an independent fork to provide Debian packaging. Consider checking out the upstream repo, https://github.com/modelon-community/fmi-library, for ways of obtaining newer versions of the library.

Please don't close this issue. Other people may have the same question, so it's nice to have it visible.

**Comment by lewismc** [comment]
@kyllingstad thanks for the response and suggestions. 
Would you be willing to 'contribute' this repository to the upstream modelon-community? That way we can maybe revive the project and begin to publish the Debain packages for each FMU release. Thanks for your consideration.

**Comment by kyllingstad** [comment]
Absolutely! Feel free to copy the entire repo or just the `debian/` directory, whichever you/they prefer.

**Comment by lewismc** [comment]
@kyllingstad roger that. I will work it with the upstream project community. Thanks. I'll come back here with the concrete proposal ASAP.

**Comment by kyllingstad** [comment]
As a tip for anyone wishing to take this over, I used to manage it with [git-buildpackage](https://honk.sigxcpu.org/piki/projects/git-buildpackage). That is the reason for the [repo layout](https://honk.sigxcpu.org/projects/git-buildpackage/manual-html/gbp.intro.html#gbp.repository) with an `upstream` branch (and `upstream/<version>` tags) to track upstream releases, and a `master` branch (with `debian/<version>` tags) for the deb packaging. That said, it is by no means *necessary* to use git-buildpackage to continue development, the `debian/` folder contents it generates are completely standard.
